nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/ServerListener.java

```diff
@@ -195,6 +195,21 @@ void start() {
         serverThread.start();
     }
 
+    boolean hasTls() {
+        return listenerConfig.hasTls();
+    }
+
+    void reloadTls(Tls tls) {
+        if (!listenerConfig.hasTls()) {
+            throw new UnsupportedOperationException("TLS is not enabled on the socket " + socketName
```

Comment:
`UnsupportedOperationException` sound bad to me in this context. Not sure - maybe `IllegalArgumentException` is better, as the tls is not correct for this socket